backend/leaderboard/models.py

```
@@ -41,14 +41,58 @@ def has_category_contributions(user, category_slug):
+    from social_tasks.models import SocialTaskCompletion
+
+    _social_tasks_table_ready = (
+        SocialTaskCompletion._meta.db_table in connection.introspection.table_names()
     )
-    return query.aggregate(
-        total=Sum('frozen_global_points')
-    )['total'] or 0
+    return _social_tasks_table_ready
+
+
+def _social_task_category_points(user, category_slug, before_date=None):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_social_task_category_points`

(ANN202)